js/src/components/item-search.component.js

```diff
@@ -41,13 +48,19 @@ export default {
         },
     },
     methods: {
-        onChange: debounce(function (e) {
-            if (this.query !== this.temp) {
-                if (this.query === '') this.query = null;
-                this.sendQuery();
-                this.temp = this.query;
-            }
-        }, 350),
+        onChange: function () {
```

Comment:
no, this is not good, we should not have an option for it in search compoment, instead we should have one global function like debounce, with with an global possibility to override

Reply:
better to revert, we can not define so much code everywhere

Reply:
This is what it does...

Reply:
Done! You can now built a package using `npm run build-behat `which will return a bebounce value of 0 for all debounce function.

Reply:
@ibelar do not be ignorant, listen why, revert this and please produce less code in favor of maintaineable code, your productivity is very good, but you often come up with hacky solutions which you then defend like a champion, but why?

Reply:
@mvorisek - did you ever work in Vue? Reason why It need to be like this is that it allow customization of debounce value. But in order to do so, you can just pass the timeout value inside the vue method because it is out of context. 
In order to be in context and be able to use this.timeout props we need to be inside the method, thus the code you see now. But hey you can try for yourself if you want...

Reply:
I have changed to another way of setting proper `this` context. Does the same as before but cleaner.